roboflow/core/project.py

```diff
@@ -227,21 +226,27 @@ def __annotation_upload(self, annotation_path, image_id):
         # Return annotation response
         return annotation_response
 
-    def check_valid_image(self, image_path):
-        try:
-            img = Image.open(image_path)
-            valid = img.format in ACCEPTED_IMAGE_FORMATS
-            img.close()
-        except UnidentifiedImageError:
-            return False
+    def check_valid_image(self, image_path, is_hosted=False):
+        if not is_hosted:
+            try:
+                img = Image.open(image_path)
+                valid = img.format in ACCEPTED_IMAGE_FORMATS
+                img.close()
+            except UnidentifiedImageError:
+                return False
```

Comment:
```suggestion
image = Image.open(image_path).convert("RGB")
```
If the image is invalid, PIL will raise an Exception that is perfectly fine since the user will be notified